### DISCUSSION TITLE
Combining projects in one project file

### DISCUSSION THREAD

**Discussion by KerimF1** [WinMerge/winmerge#748] [Ideas]
Sometimes one may need opening repeatedly two or more specific projects.
Now, this is done manually by opening them one after another.

I wonder if, by one project file only, more than one project could be opened.
I mean; perhaps such a multi-project file could be created by a text editor by appending more projects to the first one. The program can read them all and opens them one after another.

**Comment by sdottaka** [comment]
Although there is no GUI support, you can include multiple `<paths>` tags in a project file, so you can compare multiple folders and files in one project file.

e.g.
~~~
<?xml version="1.0" encoding="UTF-8"?>
<project>
	<paths>
		<left>Compare\Dir1\</left>
		<right>Compare\Dir2\</right>
		<filter>*.*</filter>
		<subfolders>1</subfolders>
		<left-readonly>0</left-readonly>
		<right-readonly>0</right-readonly>
	</paths>
	<paths>
		<left>Compare\Dir2\</left>
		<right>Compare\Dir3\</right>
		<filter>*.*</filter>
		<subfolders>1</subfolders>
		<left-readonly>0</left-readonly>
		<right-readonly>0</right-readonly>
	</paths>
</project>
~~~

**Reply by KerimF1** [reply]
So it is implemented already.
Thank you.
